src/Components/bookHub/BookCard.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import React, { useState, useEffect } from 'react';
+import Card from '@mui/material/Card';
+import Typography from '@mui/material/Typography';
+import './BookCard.css';
+import { makeStyles } from '@mui/styles';
+import { CardActionArea, CardMedia } from '@mui/material';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+const useStyles = makeStyles({
+  title: {
+    gridRowStart: 3,
+    fontSize: 14,
+    fontWeight: 'bold',
+    paddingBottom: '5px',
+  },
+
+  author: {
+    gridRowStart: 4,
+    fontSize: 12,
+    fontWeight: 100,
+  },
+});
+
+export default function BookCard({
+  id, title, author, image,
+}) {
+  const classes = useStyles();
+
+  const [authorVar, setAuthor] = useState();
+
+  const getAuthor = () => {
+    base('Creator').find(author, (err, record) => {
+      if (err) { console.error(err); }
+      setAuthor(record);
+    });
+  };
+
+  useEffect(getAuthor, []);
+
+  return ( // horizontal scroll not implemented
+    <Card className="card" sx={{ maxWidth: 345 }}>
+      <Link class="link" to={`/book/${id}`} target="_blank">
```

Comment:
The target field in the link causes a new page to be opened! We don't particularly want that to happen, so it should be removed!